packages/app-desktop/gui/note-viewer/index.html

```diff
@@ -154,14 +155,27 @@
 			setPercentScroll(percentScroll_);
 		}
 
-		// Note that this function keeps track of what's been added so as not to add the same CSS files multiple times
-		// It also means that once an asset has been added it is never removed from the view, which in many case is
-		// desirable, but still something to keep in mind.
-		function addPluginAssets(assets) {
+		// This function now keeps a track of the css or javascript files loaded as well switching of notes. 
+		// Recreating of div#joplin-container-pluginAssetsContainer on loading a new note ensures that plugin
+		// assets belonging to current note are only loaded into view to avoid css styling conflict issue.
+		function addPluginAssets(options) {
+			const assets = options.pluginAssets;
 			if (!assets) return;
 
 			const pluginAssetsContainer = document.getElementById('joplin-container-pluginAssetsContainer');
-
+			
+			let joplinContainerBody;
```

Comment:
That doesn't need to be defined outside the `if` block. We prefer to use `const` as much as possible and you can do that if you define the variable only where it's used.

Reply:
Thanks for the guidance, this would fixed in my next commit.